packages/design-system-tailwind-preset/README.md

```diff
@@ -0,0 +1,67 @@
+# `@metamask/design-system-tailwind-preset`
+
+Design system Tailwind CSS preset for MetaMask projects
```

Comment:
```suggestion
Design System Tailwind CSS preset for MetaMask projects
```

Make sure to update in `package.json` as well

Reply:
Updated https://github.com/MetaMask/metamask-design-system/pull/41/commits/9e08c779c7f9cc64c0ed7054d62d92631a661762